cpp/include/Ice/LocalExceptions.h

```diff
@@ -549,7 +547,7 @@ namespace Ice
         std::shared_ptr<std::string> _id;
     };
 
-    /// This exception is raised if the {@link Communicator} has been destroyed.
+    /// This exception is raised if the Communicator has been destroyed.
```

Comment:
This PR is focuses on `@link`. We need more updates to these doc-comments, in separate PRs.